### DISCUSSION TITLE
Simplifying Layer Manager

### DISCUSSION THREAD

**Discussion by omergr8** [GrapesJS/grapesjs#4041] [Q&A]
Hi, I need to know if it's possible to simplify layer manager?. Right now there's too much layers showing in layer manager, I just want to hide specific root element and make layer manager simplify.
![image](https://user-images.githubusercontent.com/62819343/147849608-85f3a885-0111-4d87-9186-721996e68310.png)
In the Image you can see how much nesting layers are showing. Right now its showing body-section wrapper-section-section container-text but I want it to be simplify as Body-section-text. Any suggestion?

**Comment by aliibrahim123** [comment]
you can use `editor.LayerManager.setRoot(yourComponent)`
and to disable it ``editor.LayerManager.setRoot(editor.getWrapper() )``

**Comment by rmadeiraneto** [comment]
@omergr8  You can set your components to be or not visible in the layers panel by changing it's prop `layerable`. If it is set to `false` it won't be shown in the layers panel.

**Comment by ronaldohoch** [comment]
Also, this discussion could help: https://github.com/artf/grapesjs/discussions/4368
